lib/meta_tags/text_normalizer.rb

```diff
@@ -45,7 +45,7 @@ def self.helpers
     # @return [String] string with no HTML tags.
     #
     def self.strip_tags(string)
-      helpers.strip_tags(string)
+      helpers.strip_tags(string).gsub('&amp;', '&')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.